package.json

```diff
@@ -25,7 +25,7 @@
     "start": "skyux serve"
   },
   "peerDependencies": {
-    "@blackbaud/skyux": "2.0.0-rc.12"
+    "@blackbaud/skyux": "2.0.0"
```

Comment:
I think we're up to 2.0.1 now...

Reply:
Yeah, I think that got released shortly after I made this PR. :P